views/js/controller/runtime/testRunner.js

```diff
@@ -256,7 +265,36 @@ define(['jquery', 'lodash', 'spin', 'serviceApi/ServiceApi', 'serviceApi/UserInf
 			    }
 			}
 		},
+        /**
+         * Mark apropriate timer by warning colors and show feedback message
+         * @param {object} cst - Time constraint
+         * @returns {undefined}
+         */
+        timeWarning : function (cst) {
+            $('#qti-timers > .qti-timer__type-' + cst.qtiClassName).addClass('qti-timer__warning');
+
+            // Initial time more than warning time in config
+            if (cst.seconds > cst.warningTime) {
+                var hours = Math.floor(cst.warningTime / 3600),
+                    minutes,
+                    seconds,
+                    message = '';
+
+                cst.warningTime = cst.warningTime - hours * 3600;
+                minutes = Math.floor(cst.warningTime / 60);
+                seconds = cst.warningTime - minutes * 60;
+                message = '';
+
+                if (hours) message += hours + ' ' + __(hours === 1 ? 'hour' : 'hours') + ' ';
```

Comment:
Also please avoid using blocks without parenthesis. You can check your code against those jshint rules in case of doubts : https://github.com/oat-sa/tao-core/blob/develop/views/build/.jshintrc